test/views/ViewAllApplicationsPageSpec.scala

```diff
@@ -87,6 +92,18 @@ class ViewAllApplicationsPageSpec extends CommonViewSpec
     def showsPrincipalAppsHeading()(implicit document: Document) =
       elementExistsByText(document, "th", s"$principalCapitalized applications") shouldBe true
 
+    def showsTermsOfUseBlueBox()(implicit document: Document) =
```

Comment:
Might it be better to name this an "ImportantInfo" box rather than "Blue", as styling changes over time?  Similar for the Green box.

Reply:
I did consider that but it wasn't nearly as funny
Plus, its phase 1 of several phases - the wording may change before the box colours will